scripts/view/main.js

```diff
@@ -4,12 +4,25 @@
 
 // Sub-implementation of lychee -------------------------------------------------------------- //
 
-let lychee = {};
+// TODO: Find out and explain: Here we declare a global (empty) object `lychee`; we also declare one in `./main/lychee.js`. Why don't they interfere with each other? How do we end up with **one** `lychee` object which contains the properties and methods of both objects?!
```

Comment:
Note: you are in the `view` directory here. This file is used only in the standalone photo viewer, not in the main Lychee front end. That's why they don't interfere.